app/services/account_service.py

```
@@ -0,0 +1,126 @@
+
+    async def get_account_details(self, account_uuid: UUID) -> AccountInfoResponse:
+        """
+        Get account details from GW2 API and sync with database in the background.
+        If API fails, fallback to database.
+        """
+        try:
+            # 1. Try to get fresh data from GW2 API
+            account_data = await self._get_account_from_api()
+
+            # 2. Sync with DB in background
+            asyncio.create_task(self._sync_account_to_db(account_data))
```

Ruff (0.14.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)